modules/widgets/src/zoom-widget.tsx

```diff
@@ -0,0 +1,114 @@
+import {FlyToInterpolator} from '@deck.gl/core';
+import type {Deck, Viewport, Widget, WidgetPlacement} from '@deck.gl/core';
+import {h, render} from 'preact';
+
+interface ZoomWidgetProps {
+  id: string;
+  viewId?: string | null;
+  placement?: WidgetPlacement;
+  zoomInLabel?: string;
+  zoomOutLabel?: string;
+  transitionDuration?: number;
+  style?: Partial<CSSStyleDeclaration>;
+  className?: string;
+}
+
+class ZoomWidget implements Widget<ZoomWidgetProps> {
+  id = 'zoom';
+  props: ZoomWidgetProps;
+  placement: WidgetPlacement = 'top-left';
+  viewId = null;
+  viewport: Viewport;
+  deck?: Deck;
+  element?: HTMLDivElement;
+
+  constructor(props: ZoomWidgetProps) {
+    this.id = props.id || 'zoom';
+    this.viewId = props.viewId || null;
+    this.placement = props.placement || 'top-left';
+    props.transitionDuration = props.transitionDuration || 200;
+    props.zoomInLabel = props.zoomInLabel || 'Zoom In';
+    props.zoomOutLabel = props.zoomOutLabel || 'Zoom Out';
+    props.style = props.style || {};
+    this.props = props;
+  }
+
+  onAdd({deck}: {deck: Deck}): HTMLDivElement {
+    const {style, className} = this.props;
+    const element = document.createElement('div');
+    element.classList.add('deckgl-widget', 'deckgl-widget-zoom');
+    if (className) element.classList.add(className);
+    Object.entries(style).map(([key, value]) => element.style.setProperty(key, value as string));
+    const ui = (
+      <div className="deckgl-widget-button-group">
+        <Button onClick={() => this.handleZoomIn()} label={this.props.zoomInLabel}>
+          <path d="M12 4.5v15m7.5-7.5h-15" />
```

Comment:
I'd like to use rounded google symbols for icons. Can you please use these? You may want to modify how the fill is applied.

**Zoom In**
```
<svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
  <path d="M13.1029 14.8971H8.89332C8.64334 14.8971 8.43196 14.8105 8.25918 14.6373C8.08639 14.4641 8 14.2521 8 14.0014C8 13.7508 8.08639 13.5396 8.25918 13.368C8.43196 13.1963 8.64334 13.1105 8.89332 13.1105H13.1029V8.90097C13.1029 8.65161 13.1895 8.43912 13.3627 8.26348C13.5359 8.08783 13.7479 8 13.9986 8C14.2492 8 14.4604 8.08783 14.632 8.26348C14.8037 8.43912 14.8895 8.65161 14.8895 8.90097V13.1105H19.099C19.3484 13.1105 19.5609 13.1971 19.7365 13.3703C19.9122 13.5436 20 13.7555 20 14.0062C20 14.2569 19.9122 14.468 19.7365 14.6397C19.5609 14.8113 19.3484 14.8971 19.099 14.8971H14.8895V19.1067C14.8895 19.3567 14.8029 19.568 14.6297 19.7408C14.4564 19.9136 14.2445 20 13.9938 20C13.7431 20 13.532 19.9136 13.3603 19.7408C13.1887 19.568 13.1029 19.3567 13.1029 19.1067V14.8971Z" fill="#18181A"/>
</svg>
```

**Zoom Out**
```
<svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28" fill="none">
  <path d="M8.89332 14.89C8.64334 14.89 8.43196 14.8032 8.25918 14.6296C8.08639 14.4561 8 14.2437 8 13.9926C8 13.7415 8.08639 13.5299 8.25918 13.3579C8.43196 13.186 8.64334 13.1 8.89332 13.1H19.099C19.349 13.1 19.5617 13.1868 19.737 13.3603C19.9123 13.5339 20 13.7462 20 13.9974C20 14.2485 19.9123 14.46 19.737 14.632C19.5617 14.804 19.349 14.89 19.099 14.89H8.89332Z" fill="#18181A"/>
</svg>
```